src/elements/OcActionDrop.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="oc-dropdown-wrapper" ref="menu" @keyup.esc="closeHandler(true)">
+    <button
+      :aria-expanded="isOpen.toString()"
+      :class="$_ocButton_buttonClass"
+      ref="button"
+      aria-haspopup="true"
+      @click="toggleDropdown"
+      @keydown.enter.prevent="openMenuAndFocus(0)"
+      @keydown.space.prevent="openMenuAndFocus(0)"
+      @keydown.down.prevent="openMenuAndFocus(0)"
+      @keydown.up.prevent="openMenuAndFocus(-1)"
+    >
+      <slot name="button"></slot>
```

Comment:
Is there a reason why it's slot and not a prop?

Reply:
Yes, so could put texts, (labelled) icons, images (with alt-text describing the function of the button), media, or what-not in there. Or style text. Slots are more flexible, especially since these components aim to be generic.

Reply:
Text and icons can be both handled via props. Haven't thought about an image though - think it would even break the layout of button now. But yeah, you're right - slot probably suits this better.